### PR TITLE
Read functionality

### PR DESCRIPTION
**Pull request by JPD99**
Read Functionality working, all criteria are finished, there was one slight warning in the terminal.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -8,11 +8,11 @@ const Header = () => {
   <NavItem>
     <a href="/catindex">See All The Cats</a>
   </NavItem>
-  <NavItem>
+  {/* <NavItem>
```
Comment (WalkerTexas-coder) [inline src/components/Header.js]: I think these lines want to get deleted. They won't be useful in the future.

File: src/pages/CatIndex.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        render(
+            <BrowserRouter>
+            <CatIndex cats={mockCatsArray}/>
+            </BrowserRouter>
+
+
+        )
+        // testing moving branches
+        // const main = document.createElement("main")
+        //     render(<CatIndex cats={mockCatsArray} />, main)
```
Comment (WalkerTexas-coder) [inline src/pages/CatIndex.test.js]: I think this code can go too

File: src/pages/Home.js [hunk 3]
```diff
@@ -3,7 +3,10 @@ import React from 'react'
 
 const Home = () => {
     return (
+        <>
         <h1>Welcome to Cat Tinder</h1>
+        <h2>Cat Tinder is an application that brings petowners together </h2>
```
Comment (WalkerTexas-coder) [inline src/pages/Home.js]: Indentation

**Review by WalkerTexas-coder — COMMENTED** [review]
few items to clean up

File: src/components/Header.js (hunk 1 above)
Comment (JPD99) [reply src/components/Header.js]: Resolved!

File: src/pages/CatIndex.test.js (hunk 2 above)
Comment (JPD99) [reply src/pages/CatIndex.test.js]: Resolved!

File: src/pages/Home.js (hunk 3 above)
Comment (JPD99) [reply src/pages/Home.js]: Resolved

File: src/pages/CatShow.js [hunk 4]
```diff
@@ -1,8 +1,50 @@
...
+                                Button
+                                </Button>
+                            </CardBody>
+                            </Card>
+        
+        
+        
+        
+        
+        </>
```
Comment (WalkerTexas-coder) [inline src/pages/CatShow.js:45]: Clean up this white space and indentation

**Review by WalkerTexas-coder — APPROVED** [review]
Clean up the whitespace and and indentation. Good work!

**Pull request merged**